Scripts/sp_BlitzCache.sql

```diff
@@ -0,0 +1,6896 @@
+SET ANSI_NULLS ON;
```

Comment:
Is the Scripts folder intended to be manually applied to the  SQL server, or will the install of OpsServer take care of applying these?

Reply:
The script needed to be co-piloted as I was using a custom version to deal with the spaces. I've sorted that now so we don't really need it. For a proper implementation it should check if the sp exists and turn it on or off. Just haven't had time.